fuzzing/repositories.bzl

```diff
@@ -14,19 +14,17 @@
 
 """Contains the external dependencies."""
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_jar")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("//fuzzing/private/oss_fuzz:repository.bzl", "oss_fuzz_repository")
 
-def rules_fuzzing_dependencies(oss_fuzz = True, honggfuzz = True, jazzer = False):
+def rules_fuzzing_dependencies(oss_fuzz = True, honggfuzz = True, jazzer = True):
```

Comment:
Double-checking: Is it time to flip this to True now?

(The rationale for keeping it False was that the majority of use cases of fuzzing were still for C++ only, and that dual Java and C++ Bazel projects were rather rare. 

In other words, if our users upgraded rules_fuzzing to the current version, in how many cases would the extra new deps would go unused? If that's a majority of users, then I'd argue against flipping this flag.)

Reply:
This commit makes it so that Java fuzz tests "just work" with `jazzer = True`, which adds two external repos, whereas previously calling Jazzer's own WORKSPACE macros was required additionally. 

I now see the situation as follows: An additional repository with a sufficiently unique name (I pushed a commit to prefix it with `rules_fuzzing`) that is only fetched lazily when the corresponding functionality, in this case `java_fuzz_test`, is used, doesn't really have much of an associated cost. It won't conflict with user-defined deps and won't be evaluated until used.

I of course don't have reliable data on what percentage of rules_fuzzing users uses which language, but based on questions asked on Bazel Slack, I would expect Java to be at least as popular as e.g. honggfuzz, which is also a "cheap" dep in the sense I described above.

Reply:
> This commit makes it so that Java fuzz tests "just work" with `jazzer = True`, which adds two external repos, whereas previously calling Jazzer's own WORKSPACE macros was required additionally.

Good point, I had missed this aspect!

> 
> I now see the situation as follows: An additional repository with a sufficiently unique name (I pushed a commit to prefix it with `rules_fuzzing`) that is only fetched lazily when the corresponding functionality, in this case `java_fuzz_test`, is used, doesn't really have much of an associated cost. It won't conflict with user-defined deps and won't be evaluated until used.

Sound good to me!